llvmlite/tests/test_binding.py

```diff
@@ -718,13 +718,21 @@ def test_linux(self):
             self.fail("failed parsing dependencies? got %r" % (deps,))
         # Ensure all dependencies are expected
         allowed = set(['librt', 'libdl', 'libpthread', 'libz', 'libm',
-                       'libgcc_s', 'libc', 'ld-linux', 'ld64'])
+                       'libgcc_s', 'libc', 'ld-linux', 'ld64', 'libzstd',
+                       'libstdc++'])
```

Comment:
Why are these changed? Do the new libs require a change in the manylinux target platform for wheel?

Reply:
FWIW: this is not related to wheel builds, but conda packages.